roles/sushy_emulator/templates/secret_yaml.j2

```diff
@@ -5,9 +5,9 @@ metadata:
     namespace: {{ cifmw_sushy_emulator_namespace }}
 data:
     ssh-publickey: |
-        {{ _cifmw_sushy_emulator_public_key_b64.content | default('""') | trim }}
+        {{ _sushy_emulator_key.public_key | b64encode | default('""') | trim }}
     ssh-privatekey: |
-        {{ _cifmw_sushy_emulator_private_key_b64.content | default('""') | trim }}
+        {{ _sushy_emulator_private_key.content | default('""') | trim }}
```

Comment:
i see, this one does not need b64encode because it was slurped hence already b64encoded